test/integration/scheduler_perf/main_test.go

```diff
@@ -17,11 +17,38 @@ limitations under the License.
 package benchmark
 
 import (
+	"flag"
+	"fmt"
+	"os"
 	"testing"
 
+	"github.com/spf13/pflag"
+	logsapi "k8s.io/component-base/logs/api/v1"
+	"k8s.io/klog/v2/ktesting"
 	"k8s.io/kubernetes/test/integration/framework"
 )
 
 func TestMain(m *testing.M) {
+	// Run with -v=2, this is the default log level in production.
+	ktesting.DefaultConfig = ktesting.NewConfig(ktesting.Verbosity(2))
+	ktesting.DefaultConfig.AddFlags(flag.CommandLine)
+
+	c := logsapi.NewLoggingConfiguration()
+
+	// component-base only supports pflag at the moment.
+	var fs pflag.FlagSet
+	logsapi.AddFlags(c, &fs)
+	// Not ideal. https://github.com/spf13/pflag/pull/330 would be better.
+	fs.VisitAll(func(f *pflag.Flag) {
+		if flag.CommandLine.Lookup(f.Name) == nil {
+			flag.CommandLine.Var(f.Value, f.Name, f.Usage)
+		}
+	})
+	flag.Parse()
+	if err := logsapi.ValidateAndApply(c, nil /* no feature gates */); err != nil {
+		fmt.Fprintf(os.Stderr, "error: %v\n", err)
+		os.Exit(1)
+	}
```

Comment:
While you are at it, this also would be useful here:
```
	// Contextual logging is needed for per-test output.
	// This can be removed once it's beta and logsapi.ValidateAndApply no
	// longer disables it when no feature gates are provided.
	klog.EnableContextualLogging(true)
```